src/components/cc-logs-date-range-selector/cc-logs-date-range-selector.js

```diff
@@ -131,9 +133,10 @@ export class CcLogsDateRangeSelector extends LitElement {
 
   /**
    * @param {LogsDateRangeSelectOption} option
-   * @return {string}
+   * @param {boolean} withDate
+   * @return {string|TemplateResult}
    */
-  _getOptionLabel(option) {
+  _getOptionLabel(option, withDate) {
```

Comment:
nitpick: should this be renamed to a `_renderOptionLabel` since it can return a `TemplateResult`?